lexico/cli.py

```diff
@@ -52,9 +52,11 @@ def init():
         click.echo('In order to fetch information, this services requires you' \
                ' to provide a Wordnik API key.\n' \
                'Visit http://www.wordnik.com/signup to SignUp.')
-
         # Step 01: Save the API Key
         api_key = click.prompt('Enter your Wordnik API key').strip()
+        while(incorrect_key(api_key)):
+            api_key = click.prompt('API key Entered is Incorrect.Try Again.....\nEnter your Wordnik API key').strip()
```

Comment:
I think it will be better if we can split the message into a response and a prompt.

Response: `click.echo('The API key is incorrect.')
Confirmation: `click.confirm('Would you like to re-enter your API key?')` - Read about this in Click documentation.
Prompt: 'click.prompt('Re-enter your Wordnik API key')()'

Also, I think creating an infinite loop which forces the user to enter a valid API key can be a bit bullish. Let's use the `confirmation` to break out of this loop.